libs/common/src/services/account/account.service.ts

```diff
@@ -0,0 +1,29 @@
+import { ApiService } from "@bitwarden/common/abstractions/api.service";
+import { LogService } from "@bitwarden/common/abstractions/log.service";
+import { MessagingService } from "@bitwarden/common/abstractions/messaging.service";
+import { UserVerificationService } from "@bitwarden/common/abstractions/userVerification.service";
+
+import { AccountService as AccountServiceAbstraction } from "../../abstractions/account/account.service.abstraction";
+import { Verification } from "../../types/verification";
+
+export class AccountService extends AccountServiceAbstraction {
```

Comment:
While we do use abstract classes that's just to handle Angular DI framework. They should not be extended.

```suggestion
export class AccountService implements AccountServiceAbstraction {
```

Reply:
Yeah that makes a lot of sense. Fixed!